src/globalStyles.js

```diff
@@ -52,7 +52,14 @@ class GlobalStyles {
       colorPageBackground: 'rgb(45, 45, 45)',
       colorPageBackgroundTransparent: 'rgba(45, 45, 45, 0)',
       colorBorderLines: 'rgb(137, 137, 137)',
-      colorArcBackground: 'rgb(60, 60, 60)'
+      colorArcBackground: 'rgb(60, 60, 60)',
+      colorConnectionLineBad: 'rgb(255, 10, 10)',
+      colorShapeBorder: 'rgb(255, 255, 255)',
```

Comment:
Quite a few of these colors are very different than the default vizceral style. It makes sense to have different color variables for the built in shapes, but let's keep the built-in colors consistent with the theme of vizceral. You could then pass in these specific colors in your initial configuration.

Reply:
Agree, will make something like shapes.styles to avoid mixing up with the rest of the styles